src/redux/books/books.js

```diff
@@ -0,0 +1,32 @@
+/** @format */
+
+const ADD_BOOK = 'add_book';
+const REMOVE_BOOK = 'remove_book';
```

Comment:
- [ ] As it is a requirement for this project, " `Write your book's actions and reducer (in /src/redux/books/books.js) using the ducks pattern.` ", please kindly correct your actions to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux). Something like so: 
`const ADD_BOOK = bookstore/books/ADD_BOOK`,
`const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`